example/ssd/demo.py

```diff
@@ -131,13 +138,59 @@ def parse_data_shape(data_shape_str):
         raise ValueError("Unexpected data_shape: %s", data_shape_str)
     return data_shape
 
-if __name__ == '__main__':
-    args = parse_args()
-    if args.cpu:
-        ctx = mx.cpu()
-    else:
-        ctx = mx.gpu(args.gpu_id)
+def draw_detection(frame, det, class_names):
```

Comment:
now we have two versions of drawing function, one with cv2, one with matplotlib. Is it necessary?